lte/protos/mconfig/mconfigs.proto

```diff
@@ -120,14 +120,38 @@ message PipelineD {
     string sgi_management_iface_ip_addr = 14;
     string sgi_management_iface_gw = 15;
 
-    // disable_header_enrichment is true iff header enrichment feature should
-    // be disabled for gateway
-    bool disable_header_enrichment = 16;
+    message HEConfig {
+        enum EncryptionAlgorithm {
+            RC4 = 0;
+            AES256_CBC_HMAC_MD5 = 1;
+            AES256_ECB_HMAC_MD5 = 2;
+            GZIPPED_AES256_ECB_SHA1 = 3;
+        }
+        enum HashFunction{
+            MD5 = 0;
+            HEX = 1;
+            SHA256 = 2;
+        }
+        enum EncodingType {
+            BASE64 = 0;
+            HEX2BIN = 1;
+        }
+
+        // true if header enrichment feature should be disabled for gateway
```

Comment:
Here and below: prefer to follow the golang style of starting a doc comment with the name of the thing being identified. So e.g. here it would be `disable_header_enrichment is true if header ...`